packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,307 @@
+@ffe-blue-royal-escape: escape('@{ffe-blue-royal}');
+@ffe-blue-royal-fill: @ffe-blue-royal-escape;
+@ffe-blue-azure-escape: escape('@{ffe-blue-azure}');
+@ffe-blue-azure-fill: @ffe-blue-azure-escape;
+
+.ffe-file-upload {
+    margin-top: 45px;
+    display: grid;
+    max-width: 400px;
+    grid-template-columns: 1fr;
```

Comment:
Could you add a line regarding browser support as this package is using css grids?